app/jobs/sms_reply_sender_job.rb

```diff
@@ -0,0 +1,13 @@
+class SmsReplySenderJob < ApplicationJob
+  queue_as :sms
+
+  def perform(params)
+    send_reply(params)
```

Comment:
While reviewing the changes that went into RC 62, I noticed that this line is not tested. Can we add a test please?